common/src/test/java/tachyon/security/LoginUserTest.java

```diff
@@ -54,7 +54,7 @@ public void getSimpleLoginUserTest() throws Exception {
     User loginUser = LoginUser.get(conf);
 
     Assert.assertNotNull(loginUser);
-    Assert.assertFalse(loginUser.getName().isEmpty());
+    Assert.assertEquals(loginUser.getName(), System.getProperty("user.name"));
```

Comment:
`user.name` is not used in code, and just in this test. It follows up @ooq 's change in #1292. Here we want to ensure the login user is exactly the one. Previous check just ensure the use is not empty. 

Since #1292 is merged, the referred code in the test is here: https://github.com/amplab/tachyon/blob/security/common/src/test/java/tachyon/security/LoginUserTest.java#L94